frontend/packages/kubevirt-plugin/src/types/vm/index.ts

```diff
@@ -73,6 +73,26 @@ export type CPURaw =
     }
   | string;
 
+export type V1NetworkInterface = {
+  name?: string;
+  model?: string;
+  macAddress?: string;
+  bootOrder?: number;
+  bridge?: {};
+  masquerade?: {};
+  sriov?: {};
+  slirp?: {};
+};
+
+export type V1Network = {
+  name?: string;
```

Comment:
Just a question: Shouldn't there be at least one field which is not optional?

Reply:
it can be optional when it is initialized internally